src/SlashCommands.tsx

```diff
@@ -1029,15 +1029,16 @@ export const Commands = [
         args: '<message>',
         runFn: function(roomId, args) {
             return success((async () => {
-                const cli = MatrixClientPeg.get();
-                const userId = cli.getUserId();
-                const userName = userId.slice(1).split(":").slice(0, 1);
                 const isChatEffectsDisabled = SettingsStore.getValue('dontShowChatEffects');
                 if ((!args) || (!args && isChatEffectsDisabled)) {
-                    args = _t("* %(userName)s sends confetti", {userName});
+                    args = _t("sends confetti");
+                    MatrixClientPeg.get().sendEmoteMessage(roomId, args);
+                } else {
+                    MatrixClientPeg.get().sendHtmlMessage(roomId, args);
```

Comment:
`sendHtmlMessage` needs a 3rd argument of `htmlBody`
I'm not sure why you'd want to use it here

Reply:
it's for sending messages like  "/confetti congratulations!" i tried with sendMessage it didn't work.

Reply:
it should probably be `sendTextMessage`